apps/aecontract/src/aect_call_tx.erl

```diff
@@ -104,6 +123,46 @@ new(#{caller_id   := CallerId,
                           },
     {ok, aetx:new(?MODULE, Tx)}.
 
+-spec from_db_format(tx() | tuple()) -> tx().
+from_db_format(#contract_call_tx{} = Tx) -> Tx;
+from_db_format(Tuple) ->
+    case setelement(1, Tuple, db_contract_call_tx) of
+        #db_contract_call_tx{
+           caller_id        = CallerId,
+           nonce            = Nonce,
+           contract_id      = ContractId,
+           abi_version      = ABI,
+           fee              = Fee,
+           ttl              = TTL,
+           amount           = Amount,
+           gas              = Gas,
+           gas_price        = GasPrice,
+           call_data        = CallData,
+           call_stack       = CallStack
+          } ->
```

Comment:
So the absence of field `call_origin` in record `db_contract_call_tx` is used as signal for detecting old version. Ok.
I see not int/map different representation for any version field so no transformation required for any field. Ok.